extensions/guacamole-auth-cas/pom.xml

```diff
@@ -210,15 +210,15 @@
         <dependency>
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-common</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0</version>
```

Comment:
It might be easier to just remove this dependency, relying on inheriting it from guacamole-ext. Unlike projects like guacamole-example, use of guacamole-common within an extension will be to service guacamole-ext.

Reply:
Removed.